lib/matplotlib/axes/_base.py

```diff
@@ -37,6 +37,11 @@
 is_string_like = cbook.is_string_like
 is_sequence_of_strings = cbook.is_sequence_of_strings
 
+def _sorted_iteritems(a):
```

Comment:
I think that the changes in this file are un-related.  This function only get used at call time to generate the arguments passed to the `Line2D` artists from the (kind of terrifying) mess of inputs we support to `plot`.